src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -130,9 +180,20 @@ public async Task<string> ValidateContext(HostAssignmentContext assignmentContex
             HttpResponseMessage response = null;
             try
             {
-                var zipUrl = assignmentContext.ZipUrl;
+                _logger.LogInformation($"Will be using {assignmentContext.ZipUrl.EnvVar} app setting as zip url");
+
+                var zipUrl = assignmentContext.ZipUrl.Url;
                 if (!string.IsNullOrEmpty(zipUrl))
                 {
+                    // ScmRunFromPackage can be empty since it is a placeholder for the actual build zip
```

Comment:
If ony ScmRunFromPackage should be skipped then we should check for that explicitly